modules/nw-rfhe-creating-hardware-event-kind-cr.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+
+[id="nw-rfhe-hardware-event-kind-cr_{context}"]
+= Creating the HardwareEvent Kind CR
+
+To start using Hardware event proxy, install the operator and then create the following HardwareEvent.
+
+
+[NOTE]
+====
+You can only create one instance of HardwareEvent in this release.
+====
+
+[source,yaml]
+----
+    apiVersion: "event.redhat-cne.org/v1alpha1"
+    kind: "HardwareEvent"
+    metadata:
+      name: "hardware-event"
+    spec:
+     nodeSelector: {}
+     transportHost: "amqp://amq-router-service-name.amq-namespace.svc.cluster.local" <1>
+----
+
+<1> The `amqp` host that is specified delivers the events at the transport layer using the AMQP protocol.
+
+This operator needs a secret to be created to access Redfish Message Registry. The secret name and spec the operand expects are under the same namespace the operand is deployed on. For example:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: redfish-basic-auth
+type: Opaque
+data:
+  username: cm9vdA==
+  password: Y2Fsdmlu
+stringData:
+  # BMC host DNS or IP address
+  hostaddr: 123.123.123.123
+
+----
+
+Here is an example of updating nodeSelector to select a specific node:
+
+
+[source,yaml]
+----
+apiVersion: "event.redhat-cne.org/v1alpha1"
+kind: "HardwareEvent"
+metadata:
+  name: "hardware-event"
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/hw-event: ""
+  transportHost: "amqp://amq-router-service-name.amq-namespace.svc.cluster.local"
+
+----
+
+You can create a Hardware Event Kind CR to observe hardware events and faults reported in the `hw-event-proxy` logs.
+
+There can be only one Hardware Event Kind CR in a cluster.
+
+The HardwareEvent Kind must have the following properties:
```

Comment:
The `HardwareEvent` resource offers the following configuration parameters:

(create a table with column | Parameter | Description |)

Parameter: nodeSelector
Description: select the worker node where hardware events are generated.
This property is required.

Parameter: transportHost
Description: `transportHost` specifies thee AMQ host url  e.g. `amqp://amq-router-service-name.amq-namespace.svc.cluster.local`
This property is required.

Parameter: logLevel
Description:
`logLevel` sets the log level in hw-event-proxy logs. The following are available log levels in order of severity: fatal, error, warning, info, debug, trace. 
This property is optional. The default value is `debug`.

Parameter: msgParserTimeout
Description: timeout value in milliseconds for Message Parser. If a message parsing request is not responded within the timeout duration, the original hardware event message is passed to the cloud native event framework.
This property is optional. The default value is `10`.

Reply:
@jzding I've added the table to https://deploy-preview-41289--osdocs.netlify.app/openshift-enterprise/latest/networking/using-rfhe.html#nw-rfhe-hardware-event-kind-cr_using-rfhe.
I've also made your other modifications. Let me know iof any further edits are required.

If you are happy with the content you worked on with me, please add an /lgtm note. I also need a QE /lgtm for this content to be merged. Can you please ask Niv to check the content and do likewise if he is happy.